README.md

```diff
@@ -1,5 +1,21 @@
-Design System Starter
-=====================
+**First-time setup instructions:**
```

Comment:
Trying to figure out if the readme is the best place for this documentation to live. It gets sort of meta, which might be confusing?

I considered a Google Doc instead, but that'll get lost for sure.

Reply:
Yeah, I think this is the first place people will looks so it might as well live here. It can always be deleted during project setup.